src/software/new_geom/angle.h

```diff
@@ -437,6 +447,13 @@ inline constexpr double Angle::toDegrees() const
     return rads / M_PI * 180.0;
 }
 
+inline constexpr Angle Angle::mod(Angle divisor) const
+{
+    return Angle::fromRadians(toRadians() - static_cast<double>(static_cast<long>(
+                                                toRadians() / divisor.toRadians())) *
+                                                divisor.toRadians());
```

Comment:
I'm confused what the `static_cast<long>` is for here?

Reply:
I'm not sure why either. I just copied this from ```old_geom```. Do you know why @MathewMacDougall?

Reply:
If it's just copy-over then that's fine, out-of-scope for this PR, falls under the geom utils redo!

Reply:
Not sure either, sorry :man_shrugging:

Reply:
Ok, let's leave this as work for the geom util rewrite. @scveloso - please mark this comment as resolved once you've seen it.

Reply:
Probably sound kind of rounding?